src/main/host/process.c

```diff
@@ -150,7 +152,17 @@ static void _process_logReturnCode(Process* proc, gint code) {
                         ((code == 0) ? "success" : "error"), code,
                         process_getName(proc));
 
-        if(code == 0) {
+        gchar* fileName = _process_outputFileName(proc, "exitcode");
+        FILE *exitcodeFile = fopen(fileName, "we");
```

Comment:
Did you mean to use "x" for exclusive rather than "e"?

Reply:
I used `e` for the `O_CLOEXEC` flag. It's unlikely that another thread will fork+exec while we're cleaning up this process, but I don't think it will hurt.

Reply:
Ah, makes sense.